_overviews/cheatsheets/index.md

```diff
@@ -405,16 +402,18 @@ yield x * y</code></pre>
     </tr>
     <tr>
       <td><pre class="highlight"><code>Some(42)</code></pre></td>
-      <td>Construct a non empty optional value</td>
+      <td>Construct a non empty optional value.</td>
     </tr>
     <tr>
       <td><pre class="highlight"><code>None</code></pre></td>
-      <td>The singleton empty optional value</td>
+      <td>The singleton empty optional value.</td>
     </tr>
     <tr>
       <td><pre class="highlight"><code>Option(null) == None
-Option(obj.unsafeMethod)</code></pre></td>
-      <td>Null-safe optional value factory</td>
+Option(obj.unsafeMethod)</code></pre>
+      <em><strong>but</strong></em>
+      <pre class="highlight"><code>Some(null) != None</code></pre></td>
+      <td>Null-safe optional value factory.</td>
```

Comment:
Is `Some(null)` useful to mention in a cheatsheet?  Seems like a corner case.  For most types of `Option`, it's not possible to encode `Some(null)`.  The one exception is `Option[String]`, since it is nullable.  With a type signature for any other `Option` (function parameter, return type, class member, variable annotation, etc.), Scala won't compile if a user tries to assign or pass `Some(null)` to it.

Reply:
My point was to show that one should be careful not to use `Some(obj.unsafeMethod)` and use `Option(obj.unsafeMethod)` instead. Option of any ref type can have `Some(null)` assigned to it.
Do you think it will be more confusing to include it?

Reply:
Ok, good point about `AnyRef`.   I guess it is common.